CHANGES.md

```diff
@@ -3,6 +3,7 @@
 New Languages:
 
 - Added 3rd party RiScript grammar to SUPPORTED_LANGUAGES (#2988) [John C][]
+- Added 3rd party HLSL grammar to SUPPORTED_LANGUAGES (#3002) [Stef Levesque][]
```

Comment:
You need to add a link as well around line 13 so it knows where to link you to, etc...

Reply:
arg, sry, first contribution :)

Reply:
No worries.